packages/dev/s2-icon-builder/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@react-spectrum/s2-icon-builder",
+  "version": "1.0.0",
+  "bin": {
+    "transform-icons": "index.js"
+  },
+  "dependencies": {
+    "parcel-namer-icon": "^1.0.0",
+    "parcel-transformer-icon": "^1.0.0"
```

Comment:
these will need to be made public too, should we rename them to be a bit more targeted at s2 icons

Reply:
lol yes, missed this before I wrote my above comment